clients/index.js

```diff
@@ -417,3 +418,17 @@ ApplicationClients.prototype.updateKValues = function updateKValues() {
         self.serviceProxy.updateServiceChannels();
     }
 };
+
+ApplicationClients.prototype.updateKillSwitches = function updateKillSwitches() {
+    var self = this;
+    self.serviceProxy.unblockAll();
```

Comment:
do not want unblockAll().

Use the build in delta feature in remoteConfig. We actually have to listen to the event and understand current and previous state and do proper deltas.